app/navigation/houseConsignment/index/items/HouseConsignmentItemNavigator.scala

```diff
@@ -200,13 +211,58 @@ class HouseConsignmentItemNavigator(houseConsignmentMode: Mode) extends Navigato
             controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex)
         }
     }
+
+  private def netWeightPageRoute(ua: UserAnswers, houseConsignmentMode: Mode, itemMode: Mode, houseConsignmentIndex: Index, itemIndex: Index): Option[Call] =
+    (config.phase6Enabled, houseConsignmentMode, itemMode) match {
+      case (true, _, NormalMode) =>
+        Some(routes.UniqueConsignmentReferenceYesNoController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode))
+      case (false, _, NormalMode) =>
+        Some(routes.AddCustomsUnionAndStatisticsCodeYesNoController.onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode, itemMode))
+      case (_, CheckMode, CheckMode) => Some(controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex))
+    }
+
+  private def addUniqueConsignmentReferenceYesNoRoute(ua: UserAnswers, houseConsignmentIndex: Index, itemIndex: Index, itemMode: Mode): Option[Call] =
+    ua.get(UniqueConsignmentReferenceYesNoPage(houseConsignmentIndex, itemIndex)).map {
+      case true =>
+        (config.phase6Enabled, houseConsignmentMode, itemMode) match {
+          case (true, _, NormalMode) =>
+            controllers.houseConsignment.index.items.routes.UniqueConsignmentReferenceController
+              .onPageLoad(ua.id, houseConsignmentMode, itemMode, houseConsignmentIndex, itemIndex)
+          case (true, CheckMode, CheckMode) => controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex)
+        }
+      case false =>
+        (config.phase6Enabled, houseConsignmentMode, itemMode) match {
+          case (true, _, NormalMode) =>
+            controllers.houseConsignment.index.items.routes.AddCommodityCodeYesNoController
+              .onPageLoad(ua.id, houseConsignmentIndex, itemIndex, houseConsignmentMode, itemMode)
+          case (true, CheckMode, CheckMode) => controllers.routes.HouseConsignmentController.onPageLoad(ua.id, houseConsignmentIndex)
+        }
+
+    }
+
+  private def uniqueConsignmentReferenceRoute(ua: UserAnswers,
```

Comment:
Again, can assume here that phase 6 is enabled. They can't hit the page otherwise.